html2asketch/nodeTreeToSketchGroup.js

```diff
@@ -17,14 +26,14 @@ export default function nodeTreeToSketchGroup(node, options) {
     Array.from(node.children)
```

Comment:
How about, instead of bringing up assignedNodes, we just try to access them here? e.g something like

```js
const regularChildren = Array.from(node.children);
const assignedNodes = Array.from(node.assignedNodes());
const allChildren = regularChildren.concat(assignedNodes);

allChildren
   .filter(…)
   .forEach(…)
   …
```

will that work? My web components knowledge is a bit limited.

Reply:
Hey @kdzwinel thanks for the reply. I tried something similar and just tried your idea. However because the `assignedNode` is a text node, I get this error:

```
Error: Evaluation failed: TypeError: node.getBoundingClientRect is not a function
```

I agree with your point about not wanting to make changes to the DOM. I will look to see if there is an alternative solution, or how to force text nodes to have a bound 😅

Reply:
I wonder if we could use a [Range](https://developer.mozilla.org/en-US/docs/Web/API/range) for this? https://stackoverflow.com/a/28457044/7816145

We could wrap text nodes in ranges and then get the bounding client of the range? That seems less invasive than appending nodes, but I'm not sure how it would work in practice.